eng/scripts/Language-Settings.ps1

```diff
@@ -3,6 +3,8 @@ $PackageRepository = "Maven"
 $packagePattern = "*.pom"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/java-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=java%2F&delimiter=%2F"
+$IndexHtmlLoc = "index.html"
```

Comment:
Is there any case where this isn't index.html, and if there is can we change it?

Reply:
Originally, .NET points to api/index.html.

https://azuresdkdocs.blob.core.windows.net/$web/dotnet/Azure.AI.TextAnalytics/1.0.0/api/index.html

I can do double check if it can be replaced by index.html

Reply:
I'd like to have them all be index.hml if possible. This is one of those other inconsistencies that has bothered me and we have to workaround. See https://github.com/Azure/azure-sdk/blob/master/eng/scripts/Update-Release-Versions.ps1#L59 as an example were we had do workaround these being different.